projects/bp-gallery/src/components/views/archives/ArchiveEditView.tsx

```diff
@@ -248,33 +252,35 @@ const ArchiveEditView = ({ archiveId }: ArchiveEditViewProps) => {
           defaultValue={archive.paypalClient ?? ''}
           placeholder={t('archives.edit.paypal.client-placeholder')}
           id='paypal'
-          errorText='Dies ist keine valide Paypal Client-Id'
+          errorText={t('archives.edit.paypal.client-error')}
           errorFn={async value => {
             if (value === '') return false;
             const response = await fetch(`https://www.paypal.com/sdk/js?client-id=${value}`);
             return !response.ok;
           }}
           onBlur={async value => {
             if (value === '') {
-              updateForm({ paypalClient: '', dirty: true });
+              updateForm({ paypal: { paypalClient: '' }, dirty: true });
```

Comment:
This probably breaks. `updateForm` does this:
```ts
setForm(form => {
  return { ...form, ...newForm };
});
```
This means it keeps the properties you don't specify from `ArchiveForm`. However, this doesn't apply recursively to `paypal`, i. e. your `updateForm` call overwrites the `paypal` property with an object that only has the `paypalClient` property.

To solve this, you can use [`merge`](https://docs-lodash.com/v4/merge/) to change `updateForm` like this:
```ts
type PartialForm = Omit<Partial<ArchiveForm>, 'paypal'> & { paypal?: Partial<PaypalProperties> };

const updateForm = useCallback((newForm: PartialForm) => {
  setForm(form => {
    return merge({}, form, newForm);
  });
}, []);
```
`merge` does the same as `{ ...form, ...newForm }`, but recursively. To allow for the partial `paypal`, you need the adjusted type.